crates/.cargo/config.toml

```diff
@@ -1,4 +1,5 @@
 [target.thumbv7em-none-eabihf]
+linker = "./fap-lld.py"
```

Comment:
Unlike the `runner` config, I could not persuade `cargo` to accept a multi-argument string here. So I'm relying on `/usr/bin/env` existing.

If we want the speed optimisations to be available to users, we probably need to consider cleaning up and publishing the tools, so downstream users can install them and set this to e.g. `flipperzero-lld`.

Reply:
I've been thinking about how we handle required binaries like `fap-lld`. It's probably not unreasonable to ask people to `cargo install flipperzero-tools` (or `flipperzero-lld`) if they want to develop applications.

I'm not if there's a way to specify a `cargo install` dependency, but it seems like something you could check in `build.rs`.

Reply:
`cargo` has [unstable support for artifact dependencies](https://doc.rust-lang.org/cargo/reference/unstable.html#artifact-dependencies), which could be a way to use this in future. But most embedded devs are used to needing a separate tooling package, so I think that's fine to end up requiring. For our own development we can continue to use the Python wrappers for convenience (and to ensure we are using the development version of the tooling instead of any installed version).